### DISCUSSION TITLE
Posting PR comment when running locally

### DISCUSSION THREAD

**Discussion by kristiyandobrev** [danger/danger#1396] [Q&A] [answered]
Not sure if if it’s mentioned somewhere but I am asking here anyway. Is it possible to run danger locally and produce PR comment?

**Comment by orta** [comment, marked as answer]
Yep: https://danger.systems/js/guides/the_dangerfile.html#using-danger-and-faking-being-on-a-ci - I think this technique should work fine on ruby too

**Reply by kristiyandobrev** [reply]
It worked. Thanks !
